internal/auth/options.go

```diff
@@ -102,8 +102,11 @@ type Options struct {
 
 	// These options allow for other providers besides Google, with potential overrides.
 	Provider         string `envconfig:"PROVIDER" default:"google"`
+	ProviderSlug     string `envconfig:"PROVIDER_SLUG" default:"google"`
```

Comment:
Are there any foreseeable situations where the provider slug would be different from the provider, or is this mostly a logical/organisational change?

Reply:
One could think of a future where we would want to run multiple google identity providers, maybe one for division A and another for division B. In such a world, we would need a logical identifier to separate the two. Thus, we can leverage `ProviderSlug` which is a pure client-side string used to uniquely identify a specific google identity provider.